pkg/transformers/greenmask/greenmask_choice_transformer.go

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package greenmask
+
+import (
+	"errors"
+	"fmt"
+
+	greenmasktransformers "github.com/eminano/greenmask/pkg/generators/transformers"
+	"github.com/eminano/greenmask/pkg/toolkit"
+	"github.com/xataio/pgstream/pkg/transformers"
+)
+
+type ChoiceTransformer struct {
+	transformer *greenmasktransformers.RandomChoiceTransformer
+}
+
+var errChoicesEmpty = errors.New("greenmask_choice: choices must not be empty")
+
+func NewChoiceTransformer(generatorType transformers.GeneratorType, params transformers.Parameters) (*ChoiceTransformer, error) {
+	choices := []*toolkit.RawValue{}
```

Comment:
As discussed, the parameters will be of type `[]string` when coming from the yaml file, so it's at this stage that we should parse them into the `RawValue` type, which is greenmask specific.